playbooks/install_application_metrics.yml

```diff
@@ -4,5 +4,6 @@
   roles:
   - role: middleware_monitoring
     monitoring_namespace: openshift-application-metrics
```

Comment:
I initially thought we can get rid of this, but it is also mentioned in bunch of other places, so let's keep it.